src/app/courses/course-card/course-card.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="course-card">
+  <div class="course-title">
+    {{ course.description }}
+  </div>
+  <br />
+  <!--   
+    <ng-content select="course-image"
+                *ngIf="course.iconUrl"></ng-content> -->
+  @if(course.iconUrl){
+  <img width="250" alt="algular-Logo" [src]="course.iconUrl" />
+  } @else { No Image available! }
```

Comment:
i guess need to add some tag here 'No Image available!'

Reply:
Issue resolved